editor/src/components/text-editor/text-editor.spec.browser2.tsx

```diff
@@ -293,6 +293,7 @@ describe('Use the text editor', () => {
 
       await pressShortcut(editor, cmdModifier, 'b')
 
+      await wait(1000)
```

Comment:
Please can we remove this or replace it with a much shorter timeout?

Reply:
Fixed in bec60f538.